java/code/src/com/redhat/rhn/manager/kickstart/crypto/BaseCryptoKeyCommand.java

```diff
@@ -86,12 +86,7 @@ public String getType() {
      */
     public void setContents(String contentsIn) {
         if (contentsIn != null) {
```

Comment:
if you do change in getKeyString then here we should (also) check for empty string.